composites/Plugin/Shared/components/YoastButton.js

```diff
@@ -15,12 +16,14 @@ import { rgba } from "../../../../style-guide/helpers";
  */
 export function addButtonStyles( component ) {
 	return styled( component )`
```

Comment:
Does this mean this function now applies styles that are specific to `button` and `a` elements?

Reply:
Yes and it's basically the same thing we're already doing for `BaseLinkButton`.